SS14.Launcher/Models/Connector.cs

```diff
@@ -224,6 +225,15 @@ private async Task LaunchClientWrap(
     {
         var cVars = new List<(string, string)>();
 
+        var disableIncompatibleMacOS = OperatingSystem.IsMacOS();
```

Comment:
I more meant that you should just ignore the compat flag boolean if on macOS, not that you should be mutating the config from here.